_includes/banner.html

```diff
@@ -1,12 +1,22 @@
-<section class="usa-banner site-banner" aria-label="Official website of the United States government,,,,,,">
+<section
+  class="usa-banner site-banner"
+  aria-label="Official website of the United States government,,,,,,"
+>
```

Comment:
The commas here are to prevent an error caused by duplicate `aria-label`'s with the same value. This occurs on the banner guidance page due to the component preview having the same value.